### ISSUE TITLE
Could not set the training steps

### ISSUE THREAD

**Issue by wingsyuan** [marian-nmt/marian#243]
Hi, 

int run-me.sh I could not find the parameter of training steps, could you tell me where to set this?

Thanks very much!

**Comment by emjotde** [comment]
Which file exactly are we talking about?

**Comment by wingsyuan** [comment]
Hi,
@emjotde 

the link to the file is as follow:
https://github.com/marian-nmt/marian-examples/blob/336740065d9c23e53e912a1befff18981d9d27ab/wmt2017-transformer/run-me.sh

Thanks a lot!

**Comment by emjotde** [comment]
In this example we do two things, we train for a maximum number of epochs or until the model stops improving on the validation set for 5 times in a row:

```
--early-stopping 5 --after-epochs $EPOCHS
```

I you want to additionally set the exact number of updates you can use `--after-batches N` (which we should rename to `--after-updates N` eventually). 

I personally rarely use specified numbers of steps and rather rely on `--early-stopping N` to tell me when the model has converged. In practice I also use more steps for early-stopping (usually 10) and just wait it out. The preset 8 epochs in this examples might be sub-optimal and using more could give you a slightly better result.

**Comment by wingsyuan** [comment]
@emjotde  
Thanks so much.
